### PR TITLE
revise the committee and change the year of the content

### PR DESCRIPTION
**Pull request by Knight-X**
change the year and revise the committee list.

### CODE REVIEW

File: en/about.html [hunk 1]
```diff
@@ -17,19 +17,18 @@
 
 <p align="center"><img src="/static/media/uploads/About/combined_python_usergroup_logo.png" class="img_block" style="margin-top: 20px; margin-bottom: 20px;" width="90%" alt=""/></p>
 
-<p>PyCon APAC is an international event. It was initiated in 2010 in Singapore, started to travel to Tokyo, Japan in 2013, and to Taipei, Taiwan this year. For PyCon APAC being held in Taiwan this year, the local PyCon Taiwan 2014 will be halted (or PyCon APAC 2014 is effectively PyCon Taiwan 2014).</p>
+<p>PyCon APAC is an international event. It was initiated in 2010 in Singapore, started to travel to Tokyo, Japan in 2013, and to Taipei, Taiwan last year.</p>
```
Comment (ccwang002) [inline en/about.html]: "For PyCon APAC being held in Taiwan this year ... will be halted (...) " 這句還是留著，把 2014 改成 2015

**Comment by ccwang002** [comment]
By the way try remove the `en/.about.html.swp` from git versioning.

**Comment by Knight-X** [comment]
delete the .about.html.swp
revise the zh-tw about.html
restore the for apac sentences...and change to 2015

**Comment by ccwang002** [comment]
Thanks!

**Pull request merged**